src/test/java/com/google/step/youtube/DescriptionParserTest.java

```diff
@@ -334,5 +345,165 @@ public void codeWithQuotes_multipleMatches() {
         assertThat(actual, equalTo(Arrays.asList("bobat15", "COOLIRPA")));
     }
 
+
+    /*                                                                   *
+     * ============= TEST 'TO' AND 'AT' FOR AFFILATE LINKS ============= *
+     * ex. "go to https://google.com" | "50% off at https://website.com" */
+
+    private final Pattern TO_AT_LINKS_PATTERN = DescriptionParser.Patterns.TO_AT_LINKS.getPattern();
```

Comment:
Why don't declare it here vs. at the top of the field?

Reply:
+1

Reply:
Moved to declare at top of class.